src/logic/export/RectLabelsExporter.ts

```diff
@@ -57,10 +58,11 @@ export class RectLabelsExporter {
         if (imageData.labelRects.length === 0 || !imageData.loadStatus)
             return null;
 
+        const labelNames: LabelName[] = LabelsSelector.getLabelNames();
         const image: HTMLImageElement = ImageRepository.getById(imageData.id);
         const labelRectsString: string[] = imageData.labelRects.map((labelRect: LabelRect) => {
             const labelFields = [
-                labelRect.labelIndex + "",
+                _.findIndex(labelNames, {id: labelRect.labelId}) + "",
```

Comment:
dodajesz pustego stringa po co? żeby skonwertowało do stringa? jeśli tak to nie lepiej `.toString()`?